docs/modules/candid-guide/pages/candid-ref.adoc

```diff
@@ -0,0 +1,11 @@
+= Reference
+
+This section provides detailed reference information about Candid supported types and links to Candid tools and libraries for currently supported host languages.
+
+* link:candid-types{outfilesuffix}[Supported types]
+* link:candid-tools{outfilesuffix}[Tools and libraries]
```

Comment:
Sorry, I still think that putting tools under "Reference" makes no sense. A language reference is a detailed description of a language, not its tool chain. Can we change this?

Reply:
I understand your point. I think you might be interpreting "reference" here more literally than was intended. I was using it as a classification umbrella for the type of information it contains rather than to specifically mean a language reference section. While I can move it or remove it, I'd like the flexibility to use this section for other topics that fall under the broader "reference" category and not strictly to mean "language reference".

Reply:
Well, a section "reference" in a language guide naturally is the language reference. ;) Sorry for being pedantic, but the term has a pretty firmly established meaning in the context of language docs, which I'd rather stay consistent with.